Makefile.in

```diff
@@ -85,6 +85,8 @@ dirs:	$(SUBDIRS)
 		$(MAKE) -C $$dir ; \
 	done
 
+dtls_prng.o:	platform-specific/dtls_prng_*.c
```

Comment:
I think this should be 

`dtls_prng.o::  $(wildcard platform-specific/dtls_prng_*.c)`

Reply:
Updated.